### ISSUE TITLE
"CRITICAL | Failed to get remote URL: ; exit status 1"

### ISSUE THREAD

**Issue by elhertz** [gleich/gh_fsync#4]
I've been running this action for a couple weeks no problem but it stopped working as of last night. What could be the issue?

```
Run gleich/gh_fsync@v1.[2](https://github.com/elhertz/crowdin-mod-translation/actions/runs/3818931669/jobs/6496144771#step:4:3).0
/usr/bin/docker run --name b06c6d7fd00c4[3](https://github.com/elhertz/crowdin-mod-translation/actions/runs/3818931669/jobs/6496144771#step:4:4)909aae226f9035fdc8_a0[4](https://github.com/elhertz/crowdin-mod-translation/actions/runs/3818931669/jobs/6496144771#step:4:5)6[5](https://github.com/elhertz/crowdin-mod-translation/actions/runs/3818931669/jobs/6496144771#step:4:6)b --label 29050[6](https://github.com/elhertz/crowdin-mod-translation/actions/runs/3818931669/jobs/6496144771#step:4:7) --workdir /github/workspace --rm -e "HOME" -e "GITHUB_JOB" -e "GITHUB_REF" -e "GITHUB_SHA" -e "GITHUB_REPOSITORY" -e "GITHUB_REPOSITORY_OWNER" -e "GITHUB_RUN_ID" -e "GITHUB_RUN_NUMBER" -e "GITHUB_RETENTION_DAYS" -e "GITHUB_RUN_ATTEMPT" -e "GITHUB_ACTOR" -e "GITHUB_TRIGGERING_ACTOR" -e "GITHUB_WORKFLOW" -e "GITHUB_HEAD_REF" -e "GITHUB_BASE_REF" -e "GITHUB_EVENT_NAME" -e "GITHUB_SERVER_URL" -e "GITHUB_API_URL" -e "GITHUB_GRAPHQL_URL" -e "GITHUB_REF_NAME" -e "GITHUB_REF_PROTECTED" -e "GITHUB_REF_TYPE" -e "GITHUB_WORKSPACE" -e "GITHUB_ACTION" -e "GITHUB_EVENT_PATH" -e "GITHUB_ACTION_REPOSITORY" -e "GITHUB_ACTION_REF" -e "GITHUB_PATH" -e "GITHUB_ENV" -e "GITHUB_STEP_SUMMARY" -e "GITHUB_STATE" -e "GITHUB_OUTPUT" -e "RUNNER_OS" -e "RUNNER_ARCH" -e "RUNNER_NAME" -e "RUNNER_TOOL_CACHE" -e "RUNNER_TEMP" -e "RUNNER_WORKSPACE" -e "ACTIONS_RUNTIME_URL" -e "ACTIONS_RUNTIME_TOKEN" -e "ACTIONS_CACHE_URL" -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/runner/work/crowdin-mod-translation/crowdin-mod-translation":"/github/workspace" 290506:b06c6d[7](https://github.com/elhertz/crowdin-mod-translation/actions/runs/3818931669/jobs/6496144771#step:4:8)fd00c43909aae226f9035fdc[8](https://github.com/elhertz/crowdin-mod-translation/actions/runs/3818931669/jobs/6496144771#step:4:9)
2023-01-01 22:15:22.154 |  INFO    | ⚙️ Reading from configuration file
2023-01-01 22:15:22.163 | CRITICAL | Failed to get remote URL: ; exit status 1
panic: 2023-01-01 22:15:22.163 | CRITICAL | Failed to get remote URL: ; exit status 1


goroutine 1 [running]:
log.Panicln(0xc0000c3b88, 0x1, 0x1)
	/usr/local/go/src/log/log.go:365 +0xae
github.com/gleich/logoru.Critical(0xc0000c3be0, 0x3, 0x3)
	/go/pkg/mod/github.com/gleich/logoru@v0.0.0-20210604030243-2b[9](https://github.com/elhertz/crowdin-mod-translation/actions/runs/3818931669/jobs/6496144771#step:4:10)36140[11](https://github.com/elhertz/crowdin-mod-translation/actions/runs/3818931669/jobs/6496144771#step:4:12)1e/levels.go:52 +0x1d2
github.com/gleich/gh_fsync/pkg/utils.CheckErr(0xc000018280, 0x19, 0x7814c0, 0xc00000e040)
	/usr/src/app/pkg/utils/error.go:[13](https://github.com/elhertz/crowdin-mod-translation/actions/runs/3818931669/jobs/6496144771#step:4:14) +0xc5
github.com/gleich/gh_fsync/pkg/utils.RunCommand(0x723473, 0x3, 0xc0000c3d80, 0x3, 0x3, 0x7299ed, 0x18, 0x0, 0x4a65c5)
	/usr/src/app/pkg/utils/command.go:[16](https://github.com/elhertz/crowdin-mod-translation/actions/runs/3818931669/jobs/6496144771#step:4:17) +0x1f1
github.com/gleich/gh_fsync/pkg/config.rawRead(0x726e24, 0x11, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/usr/src/app/pkg/config/config.go:60 +0x[20](https://github.com/elhertz/crowdin-mod-translation/actions/runs/3818931669/jobs/6496144771#step:4:21)5
github.com/gleich/gh_fsync/pkg/config.Read(0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/usr/src/app/pkg/config/config.go:45 +0x99
main.main()
	/usr/src/app/main.go:11 +0x34
```

**Comment by gleich** [comment]
hey @elhertz, thanks for the bug report. What happens when you run `git config --get remote.origin.url` locally in the repo?

**Comment by elhertz** [comment]
> hey @elhertz, thanks for the bug report. What happens when you run `git config --get remote.origin.url` locally in the repo?

Nothing
![image](https://user-images.githubusercontent.com/13700335/210189403-7420c6b8-279f-4712-ac45-0dbc95ac4127.png)

**Comment by gleich** [comment]
lol all good, I am working on a fix rn

**Comment by gleich** [comment]
ok, this should fix the issue. Could you run `gh_fsync@master` instead of `gh_fsync@1.2.0` and see if it works?

**Comment by elhertz** [comment]
It seems to be working correctly now. Thanks!

**Comment by gleich** [comment]
awesome! The fix is now out in release v1.2.1
